Sources/swift-parser-cli/TerminalUtils.swift

```diff
@@ -0,0 +1,51 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(Glibc)
+import Glibc
+#elseif os(Windows)
+import CRT
+#else
+import Darwin.C
+#endif
+
+#if os(Android)
+typealias FILEPointer = OpaquePointer
+#else
+typealias FILEPointer = UnsafeMutablePointer<FILE>
+#endif
+
+enum TerminalHelper {
+  static var isConnectedToTerminal: Bool {
+    return isTTY(stderr)
+  }
```

Comment:
Since we are printing the errors to stdout (instead of stderr like swift-format), we should be checking whether stdout is connected to a terminal here. For example, stdout could be connected to a terminal while stderr is not if you do `swift-parser-cli … 2> /tmp/errors.txt`.

And, to be clear, we should probably name the method to mention that is checking `stdout` instead of `stderr`.

```suggestion
  static var isStdoutConnectedToTerminal: Bool {
    return isTTY(stdout)
  }
```